### PR TITLE
BTA-8264 Add XOF Mobile Collections 

### PR DESCRIPTION
**Pull request by hillarioh**
Changes
- Add XOF MOBILE COLLECTIONS
- ADD USSD_VOUCHER UX_FLOW 
- Add Instructions for XOS Mobile Collections

Screenshots

Screenshot 2022-05-23 at 07 45 37

Screenshot 2022-05-23 at 08 05 01

### CODE REVIEW

**Review by evanson — COMMENTED** [review]
Do share a screenshot of what the collections page looks like with this change.

**Review by emanuelegorga — COMMENTED** [review]
Could you please add some description of what the content you are adding is for as part of your PR description?

File: _docs/collection-details.md [hunk 1]
```diff
@@ -183,11 +200,112 @@ Once the transaction is created, instructions for completing payment will be sen
 
 Once the funds have been successfully received from the sender, `payin_method.paid_in` and `transaction.paid_in` webhooks will be sent out.
 
+# XOF mobile collections
+
+To initiate a XOF mobile collection, please use the following details (`phone_number` used below are examples):
+At the moment, we only support collections in:
+
+- Senegal(Orange) - this requires the sender to request an OTP by typing retrieve an auth code via USSD #144#391\*ORANGE_MONEY_PIN_CODE#. The auth code retrieved should be included in the `otp` parameter in `in_details`.
```
Comment (emanuelegorga) [inline _docs/collection-details.md:208]:
```suggestion
- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD #144#391\*ORANGE_MONEY_PIN_CODE#. The auth code retrieved should be included in the `otp` parameter in `in_details`.
```

**Review by emanuelegorga — COMMENTED** [review]
Please make sure this gets merged only when everything else on the API side is production ready. Perhaps, sync with Product to know when this should be merged.

**Review by emanuelegorga — APPROVED** [review]

File: _docs/collection-details.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ title: Collection details
 permalink: /docs/collection-details/
 ---
 
-* Table of contents
-{:toc}
+- Table of contents
+  {:toc}
```
Comment (evanson) [inline _docs/collection-details.md:7]: @hillarioh This change broke the Table of contents and should be reverted.
Screenshot 2022-05-26 at 12 28 17

**Review by evanson — CHANGES_REQUESTED** [review]

**Comment by sztupy** [comment]
@hillarioh please have a look at the comments above and address them so this ticket can be merged

File: _docs/collection-details.md [hunk 3]
```diff
@@ -183,11 +200,112 @@ Once the transaction is created, instructions for completing payment will be sen
 
 Once the funds have been successfully received from the sender, `payin_method.paid_in` and `transaction.paid_in` webhooks will be sent out.
 
+# XOF mobile collections
+
+To initiate a XOF mobile collection, please use the following details (`phone_number` used below are examples):
+At the moment, we only support collections in:
+
+- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD #144#391\*ORANGE_MONEY_PIN_CODE#. The auth code retrieved should be included in the `otp` parameter in `in_details`.
```
Comment (sztupy) [inline _docs/collection-details.md:208]: Can we add backticks around the USSD code to make it more pop up

File: _docs/collection-details.md [hunk 4]
```diff
@@ -183,11 +200,112 @@ Once the transaction is created, instructions for completing payment will be sen
...
+  },
+  in_details: {
+    phone_number: "+221123456789",
+    mobile_provider: "orange",
+    otp: "123456"
+  },
+  out_details: {
+    "style": "info"
+  },
+  instructions: {
```
Comment (sztupy) [inline _docs/collection-details.md:257]: @evanson shouldn't we add proper instructions here and in `out_details` as well?

Comment (evanson) [reply _docs/collection-details.md:257]: The instructions are necessary **before** the API call and would therefore be redundant.

Comment (hillarioh) [reply _docs/collection-details.md:257]: working on them

File: _docs/collection-details.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: Collection details
 permalink: /docs/collection-details/
 ---
 
-* Table of contents
+- Table of contents
```
Comment (evanson) [inline _docs/collection-details.md:6]:
```suggestion
* Table of contents
```

File: _docs/collection-details.md [hunk 6]
```diff
@@ -183,11 +200,113 @@ Once the transaction is created, instructions for completing payment will be sen
 
 Once the funds have been successfully received from the sender, `payin_method.paid_in` and `transaction.paid_in` webhooks will be sent out.
 
+# XOF mobile collections
+
+To initiate a XOF mobile collection, please use the following details (`phone_number` used below are examples):
+At the moment, we only support collections in:
+
+- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD `#144#391\*ORANGE_MONEY_PIN_CODE#` . The auth code retrieved should be included in the `otp` parameter in `in_details`.
```
Comment (evanson) [inline _docs/collection-details.md:208]:
```suggestion
- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD `#144#391*ORANGE_MONEY_PIN_CODE#` . The auth code retrieved should be included in the `otp` parameter in `in_details`.
```

**Review by evanson — APPROVED** [review]

**Pull request merged**